packages/common/src/components/icons/Back.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
ooooh so that's where we keep our icons. I had no idea, sorry fam 😅

Reply:
Honestly found that now too ahah

Reply:
Haha, I added it a couple days ago 😄.